app/src/main/java/com/android/wildex/model/social/Comment.kt

```diff
@@ -10,12 +10,20 @@ import com.google.firebase.Timestamp
  * @property postId The ID of the post that this comment belongs to.
  * @property authorId The ID of the user who made the comment.
  * @property text The content of the comment.
- * @property date The date and time when the comment was made.
+ * @property date The date and time when the comment was made. //@property tag The tag of the
+ *   comment.
```

Comment:
Seems like the // are a typo here. This is cleaner:

```suggestion
 * @property date The date and time when the comment was made. 
 * @property tag The tag of the comment.
```